packages/plazo_mx/presentation/lib/core/ratio/sizer_ratio.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+const int kMockupHeight = 667;
+const int kMockupWidth = 375;
```

Comment:
?

Reply:
those numbers are width and height of design in the figma